src/vs/platform/contextview/browser/contextViewService.ts

```diff
@@ -35,7 +35,7 @@ export class ContextViewService extends Disposable implements IContextViewServic
 
 	showContextView(delegate: IContextViewDelegate, container?: HTMLElement, shadowRoot?: boolean): IDisposable {
 		if (container) {
-			if (container !== this.container) {
+			if (container !== this.container || true) {
```

Comment:
Looks like included by mistake?

Reply:
this was a bug with the Context View that Steven and I ran into when working with shadowDOM and the context keys. This lets us set `shadowRoot?: boolean` to false in codeActionMenu. 

Not sure if I should just edit the entire statement and remove the if statements, though

Reply:
Yes making this always true seems wrong. If it the check really isn't helpful, you can remove it. But I suspect it was added for a reason

If the shadow dom is the issue, could you use https://developer.mozilla.org/en-US/docs/Web/API/Element/shadowRoot to get the element you are after

Reply:
@sbatten we added this when we were debugging - you mentioned that the check for `container` and `this.container` were always the same? If so, then we can just remove that inner if statement.

Reply:
@mjbvz @justschen opened a PR that will fix this. once merged, you should bring it into your branch and then you won't need this change.

Reply:
@sbatten still having that weird issue with the container rendering in the shadowroot https://github.com/microsoft/vscode/issues/155780